sympy/physics/mechanics/tests/test_method.py

```diff
@@ -0,0 +1,5 @@
+from sympy.physics.mechanics.method import _Methods
+from sympy.testing.pytest import raises
+
+def test_method():
+    raises(TypeError, lambda: _Methods())
```

Comment:
Not sure what the purpose of this is.

Reply:
To make sure `_Method` class is working as desired.

Reply:
So "as desired" means that if someone tries to use the class by itself a type error should be raised?

Reply:
Yes